libraries/botbuilder-adapters-slack/botbuilder/adapters/slack/slack_adapter.py

```diff
@@ -32,10 +33,12 @@ def __init__(
         self,
         client: SlackClient,
         on_turn_error: Callable[[TurnContext, Exception], Awaitable] = None,
+        options: SlackAdapterOptions = None,
     ):
         super().__init__(on_turn_error)
         self.slack_client = client
         self.slack_logged_in = False
+        self.options = options if options else SlackAdapterOptions()

```

Comment:
very small pet peeve but this could be changed to:
`self.options = options or SlackAdapterOptions`